### PR TITLE
Fixes #17858: Add a Rudder by example for using Rudder server/relays as file mirror (for repos)

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/17858

### CODE REVIEW

File: src/rudder-by-example/modules/ROOT/pages/system/file-server-relay.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
+= Using Rudder policy server as generic file server
+
+As all Rudder policy servers use an HTTP server, it is possible to use them as
+standard file servers, which can be useful for package repositories for example.
+
+Coupled with a directive to synchronize those files from the central server, it is an easy way to distribute files on a multi-site infrastructure.
```
Comment (Fdall) [inline src/rudder-by-example/modules/ROOT/pages/system/file-server-relay.adoc:6]: root

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**